docs/advanced/middleware.md

```
@@ -332,6 +332,179 @@ const budget: ChatMiddleware = {
+          Approve plan
+        </button>
+      ) : null}
+    </>
+  )
+}
+```
+
+`onInterruptResolution` does not run in the `chat()` call that paused. It
+runs once at the start of the next `chat()` call, after the client answers.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)